core/src/main/scala/kafka/utils/ZkUtils.scala

```diff
@@ -635,6 +635,30 @@ class ZkUtils(val zkClient: ZkClient,
     zkClient.deleteRecursive(path)
   }
 
+  def subscribeDataChanges(path: String, listener: IZkDataListener): Unit = {
+    zkClient.subscribeDataChanges(path, listener)
+  }
+
+  def unsubscribeDataChanges(path: String, dataListener: IZkDataListener): Unit = {
+    zkClient.unsubscribeDataChanges(path, dataListener)
+  }
+
+  def subscribeStateChanges(listener: IZkStateListener): Unit = {
+    zkClient.subscribeStateChanges(listener)
+  }
+
+  def subscribeChildChanges(path: String, listener: IZkChildListener): java.util.List[String] = {
```

Comment:
We can do that but, than we need to convert back to a java list here:
https://github.com/apache/kafka/blob/trunk/core/src/main/scala/kafka/consumer/ZookeeperTopicEventWatcher.scala#L44
Also I don't clearly understand what is happening here. The two given parameters are unused. 
Maybe I can return with a `Seq[String]` as you suggested and use an empty string and an empty List as a parameter for the handleChildChange method.